### PR TITLE
Feature#19/generate report places

### LINKED ISSUE #19: Generate places report with redux, function and dispatch
# REQUIREMENTS
- Get the report of the places using redux where the users have been in list form
- Show how many users have shown symptoms
- How many users have had covid
- How many positives and negatives

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like. 
Captura de Pantalla historial places

# Acceptance Criteria
List all required acceptance criteria
- [ ] Match the scanned QR with the QR of the place
- [ ] Once scanned show the list of places
- [ ] Good quality code

### PR DESCRIPTION
**Pull request by GracielaJazmin**
# Description
A report is generated where the places are shown on the history screen, these places are obtained from the API provided, as well as the implementation of the provided design.

### Screenshot
Captura de Pantalla 2021-04-26 a la(s) 19 26 37

### CODE REVIEW

File: AlertaCovid/src/redux/reducers/PlaceReducer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+function PlaceReducer(state = initialState, action) {
+  switch (action.type) {
+    case NEW_PLACE:
+      return {
+        ...state,
+        places: action.payload,
+      };
+    default:
+      // need this for default case
```
Comment (fernandoamz) [inline AlertaCovid/src/redux/reducers/PlaceReducer.js:14]: delete this comment

**Review by fernandoamz — APPROVED** [review]
LGTM!

**Review by JorgeGarciaRa — APPROVED** [review]
good job !!!

**Review by HRuvalcabaP — APPROVED** [review]
Good job!

**Pull request merged**